### ISSUE TITLE
Allow the world to create a Pull Request for adding a sub domain

### ISSUE THREAD

**Issue by florent-pasquer-needone** [long-term-support/welcome#1]
It will be nice to be able to create a Pull Request for adding a sub-domain.

_Exemple :_ 

PR for adding the sub-domain `stade-rennais.long-term.support`

**Comment by williamdes** [comment]
I agree, I will have to setup a GitHub action for that and add the secrets to this repo.

**Comment by florent-pasquer-needone** [comment]
Why not to permit this directly  (now) for the end-user ?

The internal logic (automation or not) is a 2nd phasis :)

**Comment by williamdes** [comment]
Well the fact is that I can not give access to editing our DNS and other settings on my Cloudflare dashboard.

But I can have a list of subdomains or a domain mapping in a JSON file and allow anyone to edit it here.
After approval a GitHub action would apply the changes on the DNS table.

Does this solution look good ?
